src/Plugin.php

```diff
@@ -264,4 +277,30 @@ public static function renderTemplate(array $template_subpathnames, array $varia
     }
   }
 
+  /**
+   * Cron event callback to ensure proper database indices.
+   */
+  public static function cron_ensure_indices() {
+    self::ensureIndex('options', 'autoload', '(autoload)');
```

Comment:
```suggestion
    self::ensureIndex('options', 'autoload', 'autoload');
```
For simplicity and readability, I'd suggest to remove the outer parentheses, as they are always required around the definition, and the possible final index options and copying/locking options are seldomly/never used:
https://dev.mysql.com/doc/refman/8.0/en/create-index.html#create-index-options